docs/guides.md

```diff
@@ -1061,213 +1057,244 @@ The same procedure may be used to install custom KEK, db, or dbx certs.
+#### VMs that won't boot due to a revocated certificate
 
-#### Create a new Secure Boot VM
+Installed OSes and installation media that were previously perfectly bootable in Secure Boot mode may become unbootable after the certificate that signed their binaries is revocated.
 
-This section describes how to create a new Secure Boot VM using the various XCP-ng interfaces.
+This is what happened in 2021 to **the certificates used by all Linux distributions that support Secure Boot**, due to major security flaws in `grub`.
 
-:::warning
-If the VM will be running Linux, ensure that the distribution supports installation with Secure Boot enabled prior to enabling Secure Boot for the VM on XCP-ng. If the distribution does not support installation with Secure Boot enabled, then first create the VM with Secure Boot disabled and enable it *only after following the distribution's instructions for enabling Secure Boot*.
-:::
+You are likely to have issues related to this in one of the following situations:
+* You try to install a new (Linux) VM with an installation media whose binaries have been revocated.
+* You try to enable Secure Boot on an existing (Linux) VM that was not enforcing Secure Boot previously, and this VM has signed binaries that have been revocated.
+* You manually updated the dbx on an existing (Linux) VM.
 
-##### Create a Secure Boot VM using XO
+Despite this, we still recommend that you always install the latest revocation database (`dbx`) on your pools. Not doing so renders Secure Boot useless from a security perspective, as any malicious binary signed with a revocated certificate would otherwise pass Secure Boot checks.
 
-During VM creation in XO, go to the *Advanced* section and select **uefi** as the **Boot firmware**. This will display a **Secure boot** toggle that can be clicked to enable Secure Boot.
+***If you can't boot an installation media***:
+* Check whether their exists an updated installer signed with a valid certificate. If yes, use it.
+* Else go to "It still can't boot" below.
 
-![](../assets/img/screenshots/xo_uefi_sb_create_option.png)
+***If you can't boot an existing VM***:
+* Disable Secure Boot, update the OS, follow any instructions from the OS provider related to the update of the signed binaries, power off, re-enable Secure Boot, try to boot.
+* Else go to "It still can't boot" below.
 
-#### Enable Secure Boot for an Existing UEFI VM
+***It still can't boot***:
+  * either disable Secure Boot for the VM, as its binaries are not secure anymore anyway. This can be temporary until an update brings properly signed binaries.
+  * or [install](#change-the-certificates-already-installed-on-a-vm) an older `dbx` to the VM, [downloaded from the archive of prior versions of `dbx` files](https://uefi.org/revocationlistfile/archive). Let us stress again that this brings little security over disabling Secure Boot, and above all must not give you a dangerous false sense of security.
```

Comment:
`Let us stress again that this brings little security over disabling Secure Boot`

On second reading of this, I'd soften this just a little to account for differing users' risk tolerance.

`Let us stress again that this exposes the system to risk, and therefore, we recommend that before choosing an archived `dbx` that users evaluate the vulnerabilities that their system will be exposed to by omitting the most recent revocations.  Above all, downgrading the `dbx` must not give you a dangerous false sense of security.`

I suggest this because some of the post-BootHole vulnerabilities only exposed the system to potential DoS attacks, but didn't surrender to the attacker complete system control the way that BootHole and friends did. If, theoretically, the UEFI forum were to release a set of revocations that only prevented DoS, that may well be within *some* users' risk tolerance, and downgrading the `dbx` would still prevent the more "severe" vulnerabilities like BootHole... so Secure Boot would still have value for these specific users.

This still jives with earlier in the document, because we mostly just warn against including *no* `dbx`.

Note that I don't think that any of the revocations to date would fit this (the revocations that revoked the DoS vulnerabilities also revoked more serious vulnerabilities too), but it may future-proof our document.

Reply:
Agreed.